event-svc/src/event/service.rs

```diff
@@ -227,7 +235,13 @@ impl EventService {
         items: &[ReconItem<EventId>],
         source: DeliverableRequirement,
     ) -> Result<InsertResult> {
-        let (to_insert, mut invalid) = Self::validate_events(items).await?;
+        let source_node_did = match &source {
+            DeliverableRequirement::Immediate | DeliverableRequirement::Lazy => &self.node_did,
```

Comment:
this feels weird and maybe like a layering violation?  Why would the requirements around IOD affect how you find the source of an event?  Are we combining behaviors into concepts that should be separate?

Reply:
Maybe we should just add an explicit source arg to this function?  And rename the current "source" arg to "delivery_reqs" or something like that.

Reply:
let use `informant`

Reply:
`insert_events` now just passes informant through to `validate_events`
`validate_events` now just passes informant through to `parse_discovered_event`
`parse_discovered_event`  now just passes informant through to `EventInsertable::new`
`EventInsertable::new` now just passes informant through to `EventInsertable{informant}`

There is just a lot of plumbing to get the `informant` from the [api, recon, migration, time event creator] to the sql bind call. 

We pulled this apart but there might still be a natral struct we are missing that is what we should be passing around for a event insert. Or maby we just have too many layers between [api, recon, migration, time event creator] and the storage.